aws/templates/id/id_rds.ftl

```diff
@@ -34,6 +34,23 @@
                 "Name" : "Encrypted",
                 "Default" : false
             },
+            {
+                "Name" : "GenerateCredentials",
+                "Children" : [
+                    {
+                        "Name" : "Enabled",
+                        "Default" : false
```

Comment:
Thinking default should be true? Just means we'd need to force a redeploy of anything that needs the password

Reply:
Maybe we could leave it disabled now and enable it on the first train release. Then we can do the password refreshes as part of our hardware refreshes?

Reply:
Sounds like a plan. Good to start with it off so we can deploy without breaking anything